go/oasis-test-runner/cmd/root.go

```diff
@@ -47,13 +57,30 @@ var (
 		Run:   runList,
 	}
 
-	rootFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	rootFlags       = flag.NewFlagSet("", flag.ContinueOnError)
+	TestParamsFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	TestParamsMask  = "params.%s.%s"
 
 	cfgFile string
+	numRuns int
 
-	scenarioMap      = make(map[string]scenario.Scenario)
+	Scenarios        = make(map[string]scenario.Scenario)
```

Comment:
Why is this public?

Reply:
I use `Scenarios` in `ba` code to fetch and compare all supported `oasis-test-runner` test cases, if none provided. Should I make the getter instead?

Reply:
By making `cmp` subcommand part of `oasis-test-runner`, `Scenarios` can now be private again.